src/lib/components/ListingPageLayout.svelte

```
@@ -39,7 +42,12 @@
 		<div class="mx-4 flex flex-col gap-4 py-4 sm:mx-10 sm:flex-row sm:gap-0">
 			<div class="my-auto flex flex-col">
 				<div class="flex w-full items-center align-middle">
-					<div class="flex flex-row">
+					<div class="flex flex-row items-center gap-3">
+						{#if backHref}
+							<a href={backHref} class="hover:bg-muted rounded-lg border p-2" aria-label="Go back">
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)